client/package-lock.json

```diff
@@ -0,0 +1,13712 @@
+{
```

Comment:
Please delete package-lock.json files before merging. I think this may break the Heroku build. That's what happened yesterday and the fix was removing before push.